test/unit/auth/auth.spec.ts

```diff
@@ -378,14 +378,16 @@ AUTH_CONFIGS.forEach((testConfig) => {
       });
 
       it('should be fulfilled given an app which returns null access tokens', () => {
-        getTokenStub = sinon.stub(ServiceAccountCredential.prototype, 'getAccessToken').resolves(null);
+        getTokenStub = sinon.stub(ServiceAccountCredential.prototype, 'getAccessToken')
+          .resolves(null as any);
```

Comment:
Are you able to use a specific type rather than `any` for these changes?

Reply:
We can but I don't see what benefit we get from such a change. Since we are basically testing for an invalid return value here, I think `any` is appropriate.